package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "hapi-university",
+  "license": "MIT",
+  "version": "0.0.1",
+  "description": "Hapi.js University",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:averyvery/university.git"
+  },
+  "scripts": {
+    "start": "nodemon lib/index.js --exec babel-node",
+    "test": "babel-node ./node_modules/istanbul/lib/cli cover _mocha && open coverage/lcov-report/index.html"
+  },
+  "dependencies": {
+    "babel-cli": "6.3.17",
+    "babel-preset-es2015": "6.3.13",
+    "hapi": "11.1.3",
+    "nodemon": "1.8.1"
+  },
+  "devDependencies": {
+    "istanbul": "1.0.0-alpha.2",
```

Comment:
:heart_eyes_cat: awwwesome. I'm going to set this up on a bunch of open source projects!